doc/classes/Line2D.xml

```diff
@@ -62,6 +63,7 @@
 			<argument index="0" name="i" type="int">
 			</argument>
 			<description>
+				Return the x/y position of point 'i'.
```

Comment:
Return -> Returns

Positions innately have x and y coordinates in 2D, so I don't think it's necessary to mention them. You also need to say what "i" is referring to:

    Returns point [code]i[code]'s position.

Use [code][/code] tags for highlighting variables, etc.